dnsmasq/data/run.sh

```diff
@@ -26,6 +26,24 @@ for host in $(bashio::config 'hosts|keys'); do
     echo "address=/${HOST}/${IP}" >> "${CONFIG}"
 done
 
+# Create static srv-hosts
+for srvhost in $(bashio::config 'srv-hosts|keys'); do
+    SRV=$(bashio::config "srv-hosts[${srvhost}].srv")
+    HOST=$(bashio::config "srv-hosts[${srvhost}].host")
+    PORT=$(bashio::config "srv-hosts[${srvhost}].port")
+    PRIORITY=$(bashio::config "srv-hosts[${srvhost}].priority")
+    WEIGHT=$(bashio::config "srv-hosts[${srvhost}].weight")
+
+    echo "srv-host=${SRV},${HOST},${PORT},${PRIORITY},${WEIGHT}" >> "${CONFIG}"
```

Comment:
We do not use anymore echo or sed to generate config files. Please make a template at `/us/share/tempio` and generate the config from that template on runtime with bashio. I.e. zwave-js do the same or homematics use that also often.

Reply:
Can you point me to the docs on how to use bashio templates, specifically how to use a for-loop?

Reply:
https://github.com/home-assistant/tempio